src/main/java/net/minecraftforge/fmllegacy/packs/ModFilePackResources.java

```diff
@@ -66,7 +66,7 @@ protected InputStream getResource(String name) throws IOException
     {
         final Path path = modFile.findResource(name);
         if(!Files.exists(path))
-            throw new ResourcePackFileNotFoundException(modFile.getFilePath().toFile(), name);
+            throw new FileNotFoundException("Can't find resource " + name + " at " + modFile.getFilePath().toString());
```

Comment:
Why change the exception type?

Reply:
Because the vanilla exception needs a file, which can't be obtained from what i can tell.

Reply:
Yep, you can't call .toFile() on anything that isn't convertible to a native path string (or URL?)